api/environments/mappers.py

```diff
@@ -0,0 +1,15 @@
+from typing import TYPE_CHECKING, Any
+
+from flag_engine.api.document_builders import build_environment_document
+
+if TYPE_CHECKING:
+    from environments.models import Environment
+
+
+def map_environment_to_document(environment: "Environment") -> dict[str, Any]:
```

Comment:
I think it would be good to include some documentation here to explain why we're doing this here and not in a schema?

Reply:
Yeah, guess I'm being a bit sneaky with this :)

Ultimately this interface will become part of https://github.com/Flagsmith/flagsmith/issues/2079. Is it okay to reference the issue in the function docstring or a comment?

Reply:
Done.